packages/tabs/src/styled/StyledTabs.ts

```diff
@@ -20,26 +19,74 @@ interface IStyledTabsProps {
   isVertical?: boolean;
 }
 
+const verticalStyling = ({ theme }: ThemeProps<DefaultTheme>) => {
+  return css`
+    display: table;
+
+    ${StyledTabList} {
+      display: table-cell;
+      margin-bottom: 0;
+      border-bottom: none;
+      vertical-align: top;
+    }
+
+    ${StyledTab} {
+      display: block;
+      margin-bottom: ${theme.space.base * 5}px;
+      border-bottom-style: none;
+      border-left-style: ${theme.borderStyles.solid};
+      border-left-color: transparent;
+      padding: ${theme.space.base}px ${theme.space.base * 2}px;
+      text-align: left;
+
+      &:last-of-type {
+        margin-bottom: 0;
+      }
+
+      &[data-garden-focus-visible]::before {
+        top: ${theme.space.base}px;
+        right: ${theme.space.base}px;
+        left: ${theme.space.base}px;
+      }
+
+      ${theme.rtl &&
+        `
+        margin-left: 0;
+        border-left: 0;
+        border-right-style: ${theme.borderStyles.solid};
+        border-right-color: transparent;
+        text-align: right;
+      `}
+    }
+
+    ${StyledTabPanel} {
+      margin-left: ${theme.space.base * 8}px;
+      width: 100%;
```

Comment:
I'm curious where this `width` property came from and what it's supposed to be doing?

Reply:
Great question. I think I was playing around with the table display styling and thought it was necessary. Removed.